src/query/test/test.go

```diff
@@ -0,0 +1,577 @@
+// Copyright (c) 2018 Uber Technologies, Inc.
```

Comment:
I don't know if adding the header here is correct if this is taken from prom... otherwise, add a comment to these indicating where it was taken from and linking the appropriate prom file in their repo